image_transport/include/image_transport/camera_publisher.hpp

```diff
@@ -133,7 +142,7 @@ class CameraPublisher
 
   /*!
    * \brief Publish an (image, info) pair with given timestamp on the topics associated with
-   * this CameraPublisher.
+   * this CameraPublish:er.
```

Comment:
```suggestion
   * this CameraPublisher.
```
random typo snuck in